assets/model_monitoring/components/src/model_performance_metrics/run.py

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""Entry script for Data Drift Compute Metrics Spark Component."""
+
+import argparse
+
+from shared_utilities.patch_mltable import patch_all
```

Comment:
I think this is no longer needed and already removed from the utilities. You can remove line 8 and line 15 below.